client/kv.go

```diff
@@ -310,42 +266,21 @@ func (kv *KV) getInternal(key proto.Key) (*proto.Value, error) {
 
 // Put writes the specified byte slice value to key.
 func (kv *KV) Put(key proto.Key, value []byte) error {
-	return kv.putInternal(key, proto.Value{Bytes: value})
+	return kv.Run(PutCall(key, value, nil))
```

Comment:
We might want to remove these convenience routines. They aren't much more convenient that calling `Run` and encourage serial operations.